src/common/meta/src/instruction.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub enum Instruction {
+    OpenTable {
```

Comment:
> Let's rename it to `OpenRegion` (and `CloseRegion`, too), because the failover happens in region scope. A table could have multiple regions, if we were to optimize the region failover speed, it's relatively easy to combine multiple `OpenRegion` instructions to one `OpenTable` instruction.

I rename `OpenTable` to `OpenRegion` (as well as `CloseTable`). However, I keep the `Open/CloseTableHandler`. For now, we treat all `Open/CloseRegion` as `Open/CloseTable` instructions.